bin/build_cmd.ml

```diff
@@ -1,5 +1,7 @@
 open Import
 
+let use_autorelock = true
```

Comment:
Can you add a comment about the plan for this variable, e.g. will it always be true in the developer preview and false otherwise?

Reply:
Good point, done. Added a note that if this were to be set to true on main the whole switch should just be removed.